launch_testing/test/check_env.py

```diff
@@ -0,0 +1,4 @@
+if __name__ == '__main__':
+    import os
+    assert(os.environ.get('testenv2', 'Not Set') == 'Not Set')
```

Comment:
This line should better test that the environment variable is not set: `'testenv2' not in os.environ`